src/main/java/se/kth/sda/freethinker/lecture/Lecture.java

```diff
@@ -19,7 +22,8 @@
 
     private Date startTime;
 
-
```

Comment:
I think lecture needs relationship with subject as well.

Reply:
Yes. Need to add that as well. But I guess I will do that in a separate branch.

Reply:
I noticed that, but thought maybe we are leaving it for the last. Because I believe subject's relation to lecture is required in the scheduling component.

Reply:
Good practice! I will merge it now!